Worker.Extension.Sql/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.Sql", "0.1.*-preview")]
```

Comment:
Please make a follow up issue to make this dynamically generated during build - we still don't want to have to remember to come back and update this later on when we bump version/get out of preview. And it should be pretty straightforward to do this using the pipeline variables I showed you earlier.

Reply:
https://github.com/Azure/azure-functions-sql-extension/issues/404